WorldTrotter/ConversionViewController.swift

```diff
@@ -54,20 +84,33 @@ class ConversionViewController: UIViewController{ //, UITextFieldDelegate {
     }
     //Computed Property for Celsius Temperature Measurement (Read only property - getter without setter)
     var celsiusValue: Measurement<UnitTemperature>? {
-        if let fahrenheitValue = fahrenheitValue {
+        /*if let fahrenheitValue = fahrenheitValue {
             return fahrenheitValue.converted(to: .celsius)
         } else {
             return nil
+        }*/
+        didSet{
+            updateCelsiusLabel()
         }
     }
     // Helper Functions
     func updateCelsiusLabel() {
+        
```

Comment:
Function name is confusing since it is updating both labels. 
Two different functions, one for each label would be better.